spells.yml

```diff
@@ -9778,6 +10341,80 @@ tsunami:
+  duration: Until dispelled
+  level: 8
+  material: a skull scepter worth at least 1,000 gp
+  name: Undead Horde
+  range: 300 feet
+  sources:
+    - 2D
+  tags:
+    - arcane
+    - necromancy
+undead_transformation:
+  casting_time: 1 action
+  components: V, S
+  concentration: false
+  description: |-
+    One willing creature you touch is cursed with undeath. While cursed in this way, the target experiences the following effects:
+
+    - The target is undead in addition to its normal creature type.
+    - The target has resistance to necrotic damage and immunity to poison damage and the poisoned condition.
+    - The target has darkvision out to a range of 60 feet. If the target already had darkvision, its range is increased by 30 feet instead.
+    - At the end of every hour, the target's hit point maximum decreases by 1. This reduction can't be prevented in any way, and is removed only when the curse is lifted.
+
+    A _remove curse_ spell cast on the target ends this effect.
+  duration: Instantaneous
+  level: 4
+  name: Undead Transformation
+  range: Touch
+  sources:
+    - 2D
+  tags:
+    - arcane
+    - curse
+    - necromancy
+unreality:
+  casting_time: 1 major action
+  components: V
+  concentration: true
+  description: |-
+    You alter reality on a massive scale. This spell creates a 1,000-foot-radius sphere centered on you and remaining fixed in place for the duration. If you leave the area, the spell ends.
+
+    When you cast the spell, and as a major action on subsequent turns, you can reshape the appearance of reality within the sphere in each the following ways:
+    - All illusions and illusion spells in the area are suppressed, except for those you choose not to suppress.
+    - You can create illusory terrain, objects, and environmental details, but not creatures. The illusions can include visual, audible, and even olfactory elements. For example, you could make a burning forest with the smell of smoke in the air, but not a dragon flying through the sky.
+    - You can cause real terrain, unattended objects, and environmental details to become invisible, inaudible, and unsmellable.
+    - You can change the perceived temperature, but not enough to cause or prevent damage.
+    - You can change the light and to what degree the area is obscured.
+    - From within the sphere, you can cause the outside to appear however you wish, and vice versa. For example, you could make a small town disappear to the outside world, while the town inhabitants experience a solar eclipse.
+
+    These changes last until the spell ends or until you change them again. You can be as specific or as general as you want when making these changes; the spell automatically fills in missing details to match your proscribed reality.
+
+    If the spheres of two or more castings of _unreality_ would overlap, each point in space is controlled by the sphere whose center is closest to that point.
```

Comment:
Minor question, but what would happen to a point equidistant from the centers of the spheres?

Reply:
I wish I could just say "partition space like a Voronoi diagram" but from but the intent is that those equidistant edges don't z-fight, they take the appearance dictated by the side you're on